README.md

```diff
@@ -85,7 +86,7 @@ Then go ahead and start the job server using the instructions above.
 
 Let's upload the jar:
 
-    curl --data-binary @job-server-tests/target/job-server-tests-$VER.jar localhost:8090/jars/test
+    curl --data-binary @job-server-tests/target/scala-2.10/job-server-tests-$VER.jar localhost:8090/jars/test
```

Comment:
@lossyrob thanks for the PR!   Yeah it's probably an oversight, a while back the path changed.